README.md

```diff
@@ -100,7 +100,78 @@ To launch VM image, use the following script:
 sudo yocto/meta-aos-vm/scripts/run_aos_vm.sh output/image
 ```
 
-Follow the script output instruction to attach to nodes consoles.
+Follow the script output instruction to attach to nodes consoles. -->
+
+## Create VM Image Archive
+
+To create an archive of the VM images, use the following command:
+
+```sh
+yocto/meta-aos-vm/scripts/aos_vm.sh archive -m -s <number_of_secondary_nodes> -o <output_path>/aos-vm.tar.gz
+
+options:
```

Comment:
options explanation should be out of sh code block.

Reply:
Done